<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-12-31T10:58:40Z" name="Default suite" finished-at="2022-12-31T10:58:45Z" duration-ms="5432">
    <groups>
    </groups>
    <test started-at="2022-12-31T10:58:40Z" name="Default test" finished-at="2022-12-31T10:58:45Z" duration-ms="5432">
      <class name="testcase.TestC1">
        <test-method is-config="true" signature="openB()[pri:0, instance:testcase.TestC1@27406a17]" started-at="2022-12-31T10:58:38Z" name="openB" finished-at="2022-12-31T10:58:40Z" duration-ms="1577" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openB -->
        <test-method signature="creataccount()[pri:0, instance:testcase.TestC1@27406a17]" started-at="2022-12-31T10:58:40Z" name="creataccount" finished-at="2022-12-31T10:58:45Z" duration-ms="5392" status="FAIL">
          <exception class="java.lang.IllegalArgumentException">
            <message>
              <![CDATA[Keys to send should be a not null CharSequence]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException: Keys to send should be a not null CharSequence
at org.openqa.selenium.remote.RemoteWebElement.sendKeys(RemoteWebElement.java:97)
at testcase.pageobj.fillDetails(pageobj.java:28)
at testcase.TestC1.creataccount(TestC1.java:12)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- creataccount -->
        <test-method is-config="true" signature="closeB()[pri:0, instance:testcase.TestC1@27406a17]" started-at="2022-12-31T10:58:45Z" name="closeB" finished-at="2022-12-31T10:58:46Z" duration-ms="196" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeB -->
      </class> <!-- testcase.TestC1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
